### PR TITLE
Upload Cao_BCO

### PR DESCRIPTION
**Pull request by cderek7**
Ready for review

### CODE REVIEW

**Comment by cderek7** [comment]
Thank you !!

**Comment by cderek7** [comment]
Feel free to tell me if there are any other problems!

File: Student_BCOs/Derek_Cao_BCO 000534.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                "contribution": [
+                    "authoredBy",
+                    "derivedFrom"
+                ],
+                "email": ""
+            }
+        ]
+    },
+    "usability_domain": [
+        "Adapted from Tiangong Lu, Armand Bankhead, III,  Mats Ljungman, Nouri Neamati. \"Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3.\" PMC6735387(July 28, 2019). https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6735387/",
```
Comment (jpat1546) [inline Student_BCOs/Derek_Cao_BCO 000534.json:80]: Great job including "Adapted by [publication]". However the rest of the usability domain needs work. The usability domain should be your own summary of the paper, not sentences copied and pasted from the publication's abstract. 

From paper: 

> We used multi-omic genome-wide profiling to identify multi-level (Bru-Seq, RNA-Seq, and MS Proteomic) expression signatures of STAT3 KO ovarian cancer cells.

>We confirmed multiple previously identified STAT3-associated genes and, more importantly, identified new gene sets and 41 genes differentially regulated in response to STAT3 KO involved in a wide range of cellular processes.

>We identify a link between STAT3 and E2F target genes as well as alteration in STAT family members upon STAT3 deletion

>In this study, a combination of CRISPR-Cas9-mediated gene KO and multi-omic genome-wide profiling enabled us to identify multi-level (Bru-Seq, RNA-Seq, and MS Proteomic) expression signatures of parental and STAT3 KO ovarian cancer cells

**Review by jpat1546 — CHANGES_REQUESTED** [review]
@cderek7 
I've reviewed your updated BCO and only the usability domain remains to be fixed.Great job with those modifications- of the 9 issues assigned to you, only 1 is left to fix.

. Please make all the changes in the BCO Editor, download, and update this pull request file with the new JSON (copy/paste- same process as before). Once all changes have been made, assign this pull request back to me and I will review again. Until then, assigning the pull request back to you.

**Comment by jpat1546** [comment]
closed by #250

**Pull request closed without merging**